docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: "3"
+
+# SPECIAL NOTES:
+# 1.  GEM_HOME is set to the tmp/gem_home directory in this app.
+#     Bundler is installed here.
+#     If you delete Bundler from this directory, you will get an error
+#     message about not having the correct version of bundler when you
+#     run "bundle install" or try to do just about anything else.
+# 2.  BUNDLE_PATH is set to the tmp/bundle directory in this app.
+#     The products of "bundle install" are saved here.  If you delete
+#     this directory, subsequent executions of the "bundle install"
+#     process will take longer.
+
+services:
+
+  mailcatcher:
+    image: yappabe/mailcatcher
+    ports:
+      - "1025:1025"
+      - "1080:1080"
+
+  database:
+    image: postgres
+    env_file:
+      - .docker-env/development/database
+    volumes:
+      - db_data:/var/lib/postgresql/data
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
+    ports:
+      - "15432:5432"
+
+  rails:
+    container_name: pie_rails
+    build: .
+    volumes:
+      - .:/home/winner/myapp
+    environment:
+      - BUNDLE_PATH=/home/winner/myapp/tmp/bundle
```

Comment:
Using the tmp/bundle directory as BUNDLE_PATH means that the "bundle install" process is much quicker after you run the "docker/nuke" or "docker/nukec" command because the "bundle install" products are saved in the host environment.  This speeds up the debugging/troubleshooting process, which has been essential for working on dockerization.

Reply:
Ya! this was more about changing the `winner` name, so if you go with the username suggestions above, that's what this was about.  Sorry for the confusing comment :)